src/Microsoft.TestPlatform.Extensions.HtmlLogger/Microsoft.TestPlatform.Extensions.HtmlLogger.csproj

```diff
@@ -44,5 +44,13 @@
     <PropertyGroup Label="Configuration">
         <RootNamespace>Microsoft.VisualStudio.TestPlatform.Extensions.HtmlLogger</RootNamespace>
     </PropertyGroup>
+    <ItemGroup>
+        <!-- API that is common to all frameworks that we build for. -->
+        <AdditionalFiles Include="PublicAPI/PublicAPI.Shipped.txt" />
+        <AdditionalFiles Include="PublicAPI/PublicAPI.Unshipped.txt" />
+        <!-- Framework specific API -->
+        <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Shipped.txt" />
```

Comment:
Instead of adding empty PublicAPI* files for all targets, should we add `Condition="!Exists('PublicAPI/$(TargetFramework)/PublicAPI.Shipped.txt')"` here remove those files?

Reply:
We could but I already did the work of adding the empty files, and while they are not strictly needed, if you would want to add framework specific api you can do it right now without having to decipher the csproj, and adding files. The roslyn analyzer also has a code fix, which will output into the specific file for the tfm. So I would vote for keeping it as is.

Reply:
**they are not strictly needed** for most of the projects, because most of them (luckily) does not have tfm specific api.